tinyagentos/routes/hub.py

```
@@ -109,3 +147,356 @@ async def put_own_profile(
+        json.loads(upstream.body) if upstream.body else {},
+        status_code=upstream.status_code,
+        media_type=upstream.media_type,
+    )
+
+
+@router.post("/api/hub/friends/requests/{rid}/accept")
+async def accept_friend_request(
+    rid: str,
+    body: AcceptIn,
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)